### ISSUE TITLE
hybrid loss of AnatomyNet

### ISSUE THREAD

**Issue by yangzhikai** [wentaozhu/AnatomyNet-for-anatomical-segmentation#8]
Only Dice loss can be seen in the AnatomyNet. Where is Focal loss?  Dose it mean AnatomyNet is fine-tuned  from 'baselineSERes18Conc' by using  Dice loss only?

**Comment by wentaozhu** [comment]
I just add the focal loss in https://github.com/wentaozhu/AnatomyNet-for-anatomical-segmentation/commit/fa736bf017d9ba2c12cecb76ccb7d873194c2ad0

It can be easily obtained from cross entropy loss.
